### DISCUSSION TITLE
How to execute a shortcut using Google Assistant?

### DISCUSSION THREAD

**Discussion by M1K3-NL** [Waboodoo/HTTP-Shortcuts#372] [Q&A]
Hello!
I'd like to execute a shortcut using Google Assistant.
In example if my shortcut is named "Execute Open Browser" and I say "Hey Google, Execute Open Browser" that it will run the shortcut.

**Comment by Waboodoo** [comment]
Hello.
I'm not using Google Assistant myself, so I don't have much info on what is possible there. From the side of the app, there is currently no integration with it, so I would assume that you can't really do anything useful there.
I'll try looking into this when I have the time. It would certainly be a good addition to the app and has been asked about before.

**Reply by Waboodoo** [reply]
As of version 3.8.0, the app has basic integration with Google Assistant. It's still in an experimental phase and may not fully work. You can activate it by editing a shortcut and in the "Trigger & Execution Settings" section enabling the checkbox at the top. This will make the app advertise your shortcut to Google Assistant, allowing it to be triggered from there either by name, or by "Run [name]" or "Execute [name]"
